TRAX_BOUNDED_OVERLAP: true 
TRAX_REGION_LEGACY: false 
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcublas.so.8.0 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcudnn.so.5 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcufft.so.8.0 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcuda.so.1 locally
I tensorflow/stream_executor/dso_loader.cc:135] successfully opened CUDA library libcurand.so.8.0 locally
@@TRAX:hello "trax.image=path" "trax.region=rectangle" "trax.version=1"
@@TRAX:initialize "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/sequences/ball1/00000001.jpg" "496.000000,419.000000,40.000000,42.000000" 
@@TRAX:status "496.0,419.0,40.0,42.0"
@@TRAX:frame "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/sequences/ball1/00000002.jpg" 
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE3 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.1 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use SSE4.2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use AVX2 instructions, but these are available on your machine and could speed up CPU computations.
W tensorflow/core/platform/cpu_feature_guard.cc:45] The TensorFlow library wasn't compiled to use FMA instructions, but these are available on your machine and could speed up CPU computations.
I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 0 with properties: 
name: GeForce GTX TITAN X
major: 5 minor: 2 memoryClockRate (GHz) 1.076
pciBusID 0000:0a:00.0
Total memory: 11.92GiB
Free memory: 11.71GiB
W tensorflow/stream_executor/cuda/cuda_driver.cc:590] creating context when one is currently active; existing: 0x4cf3110
I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 1 with properties: 
name: GeForce GTX TITAN X
major: 5 minor: 2 memoryClockRate (GHz) 1.076
pciBusID 0000:09:00.0
Total memory: 11.92GiB
Free memory: 11.81GiB
W tensorflow/stream_executor/cuda/cuda_driver.cc:590] creating context when one is currently active; existing: 0x4cf6f00
I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 2 with properties: 
name: GeForce GTX TITAN X
major: 5 minor: 2 memoryClockRate (GHz) 1.076
pciBusID 0000:06:00.0
Total memory: 11.92GiB
Free memory: 11.81GiB
W tensorflow/stream_executor/cuda/cuda_driver.cc:590] creating context when one is currently active; existing: 0x4cfad20
I tensorflow/core/common_runtime/gpu/gpu_device.cc:885] Found device 3 with properties: 
name: GeForce GTX TITAN X
major: 5 minor: 2 memoryClockRate (GHz) 1.076
pciBusID 0000:05:00.0
Total memory: 11.92GiB
Free memory: 11.61GiB
I tensorflow/core/common_runtime/gpu/gpu_device.cc:906] DMA: 0 1 2 3 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 0:   Y Y Y Y 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 1:   Y Y Y Y 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 2:   Y Y Y Y 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:916] 3:   Y Y Y Y 
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:0) -> (device: 0, name: GeForce GTX TITAN X, pci bus id: 0000:0a:00.0)
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:1) -> (device: 1, name: GeForce GTX TITAN X, pci bus id: 0000:09:00.0)
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:2) -> (device: 2, name: GeForce GTX TITAN X, pci bus id: 0000:06:00.0)
I tensorflow/core/common_runtime/gpu/gpu_device.cc:975] Creating TensorFlow device (/gpu:3) -> (device: 3, name: GeForce GTX TITAN X, pci bus id: 0000:05:00.0)
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
W tensorflow/core/framework/op_kernel.cc:993] Invalid argument: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/DeepKCF/DeepKCF_VOT.py", line 49, in <module>
    tracker.train(image, True)
  File "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/DeepKCF/tracker.py", line 226, in train
    self.parameters.features, self.currentScaleFactor, start)
  File "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/DeepKCF/tracker.py", line 292, in translation_sample
    self.create_tf_graph(im_patch)
  File "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/DeepKCF/tracker.py", line 317, in create_tf_graph
    self.init_fn(self.session)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/contrib/framework/python/ops/variables.py", line 586, in callback
    saver.restore(session, model_path)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/training/saver.py", line 1428, in restore
    {self.saver_def.filename_tensor_name: save_path})
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/client/session.py", line 767, in run
    run_metadata_ptr)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/client/session.py", line 965, in _run
    feed_dict_string, options, run_metadata)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/client/session.py", line 1015, in _do_run
    target_list, options, run_metadata)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/client/session.py", line 1035, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InvalidArgumentError: Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
	 [[Node: save/RestoreV2_6 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save/Const_0, save/RestoreV2_6/tensor_names, save/RestoreV2_6/shape_and_slices)]]

Caused by op u'save/RestoreV2_6', defined at:
  File "<string>", line 1, in <module>
  File "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/DeepKCF/DeepKCF_VOT.py", line 49, in <module>
    tracker.train(image, True)
  File "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/DeepKCF/tracker.py", line 226, in train
    self.parameters.features, self.currentScaleFactor, start)
  File "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/DeepKCF/tracker.py", line 292, in translation_sample
    self.create_tf_graph(im_patch)
  File "/raid/hustxly/VOT/VOT-Tracker/workspace_DKCF/DeepKCF/tracker.py", line 311, in create_tf_graph
    slim.get_model_variables('InceptionV3'))
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/contrib/framework/python/ops/variables.py", line 584, in assign_from_checkpoint_fn
    saver = tf_saver.Saver(var_list, reshape=reshape_variables)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/training/saver.py", line 1040, in __init__
    self.build()
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/training/saver.py", line 1070, in build
    restore_sequentially=self._restore_sequentially)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/training/saver.py", line 675, in build
    restore_sequentially, reshape)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/training/saver.py", line 402, in _AddRestoreOps
    tensors = self.restore_op(filename_tensor, saveable, preferred_shard)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/training/saver.py", line 242, in restore_op
    [spec.tensor.dtype])[0])
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/gen_io_ops.py", line 668, in restore_v2
    dtypes=dtypes, name=name)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/op_def_library.py", line 763, in apply_op
    op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 2327, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 1226, in __init__
    self._traceback = _extract_stack()

InvalidArgumentError (see above for traceback): Unsuccessful TensorSliceReader constructor: Failed to get matching files on /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint/inception_v3.ckpt: Not found: /tmp/tpc2a29902_bdf2_4ad5_90aa_3ead5fcb12fd/checkpoint
	 [[Node: save/RestoreV2_6 = RestoreV2[dtypes=[DT_FLOAT], _device="/job:localhost/replica:0/task:0/cpu:0"](_recv_save/Const_0, save/RestoreV2_6/tensor_names, save/RestoreV2_6/shape_and_slices)]]

Error: Unable to contact tracker.
@@TRAX:quit 
Tracker exited (exit code 11)
